test/simulation.jl

```diff
@@ -199,124 +199,6 @@ end
 Point3D = Point3{Float64}
 flatten(vfield::Function, mesh) =  ♭(mesh, DualVectorField(vfield.(mesh[triangle_center(mesh),:dual_point])))
 
-# Testing Advection-Diffusion
```

Comment:
What is wrong with these tests?

Reply:
This Advection-Diffusion sim is very old and uses a Lie Derivative that depended on primal-primal input. We've now changed to a dual-primal Lie, which is how it is described in Hirani.